src/compas/CompasAutoAlignment.ts

```diff
@@ -35,61 +43,69 @@ export default class CompasAutoAlignmentElement extends LitElement {
 
   async execute(): Promise<void> {
     if (this.valid()) {
-      await CompasSclAutoAlignmentService().updateSCL(this.doc, this.getSelectedValues())
+      await CompasSclAutoAlignmentService()
+        .updateSCL(this.doc, this.getSelectedValues())
         .then(sclDocument => {
-          this.dispatchEvent(newLogEvent({kind: 'reset'}));
+          this.dispatchEvent(newLogEvent({ kind: 'reset' }));
           this.dispatchEvent(
-            newOpenDocEvent(
-              sclDocument,
-              this.docName,
-              {detail: {docId: this.docId}}
-            ));
+            newOpenDocEvent(sclDocument, this.docName, {
+              detail: { docId: this.docId },
+            })
+          );
 
           this.dispatchEvent(
             newLogEvent({
               kind: 'info',
-              title: get('compas.autoAlignment.success')
-            }));
+              title: get('compas.autoAlignment.success'),
+            })
+          );
 
           // Close the Save Dialog.
           this.dispatchEvent(newWizardEvent());
         })
-        .catch(createLogEvent);
+        .catch(reason => createLogEvent(this, reason));
 
       // Close the Save Dialog.
       this.dispatchEvent(newWizardEvent());
     }
   }
 
   render(): TemplateResult {
```

Comment:
Is this list used anywhere?

Reply:
Yes, it's shown when you select 'Auto Align SLD' from the menu.
The user can select for which substation(s) the auto alignment should be executed.